### DISCUSSION TITLE
How do I access current time data for templates ?

### DISCUSSION THREAD

**Discussion by AlanBlanchet** [blakeNaccarato/copier-python#375] [Q&A]
Hello. I would like to add to my copier output the current date in "Y-m-d" format.
Though none of the jinja2 templating like "now" or "time.now()" work in the ".jinja" files I have.

Could you help me understand why they don't work and what I should do to get these features ?
It's not the first time that I look up some jinja code and that it doesn't work. Maybe I don't fully understand the copier eco-system ?

Many thanks.
Alan

**Comment by AlanBlanchet** [comment]
The question was a mistake. My PC took control over me and wrote it O_o

**Comment by blakeNaccarato** [comment]
No worries! I'll take a stab at it anyways. If you happened to clone/fork my template or are making your own copier template, you'll probably need to enable [certain Jinja extensions](https://copier.readthedocs.io/configuring/#jinja_extensions) (unsure why that link to Copier documentation isn't working...). Perhaps `jinja2-time` might do, as given in one of the examples.

I've stayed away from the black magic of Jinja extensions, so my template is pretty bespoke and not using lots of the latest features of Copier. Looking forward to playing with some recent features eventually as the template develops.

**Reply by AlanBlanchet** [reply]
Yes I went with `jinja2-time` and it worked as I wanted.
I didn't really know the existance of these templates and thought it was supposed to be already implemented in the copier package.
After research I found out how to do it.

Thanks for the reply ^^
